### ISSUE TITLE
ReferenceError: regeneratorRuntime in File Input

### ISSUE THREAD

**Issue by dkumor** [koumoul-dev/vuetify-jsonschema-form#301]
When using a schema which includes a file upload, I am getting the following error:
```
[Vue warn]: Error in v-on handler: "ReferenceError: regeneratorRuntime is not defined"

found in

---> <VFileInput>
       <VJsf>... (1 recursive calls)
         <VForm>
```

To the best of my understanding this happens because `vuetify-jsonschema-form` uses `babel-polyfill` for async functionality. My project does not use babel, and therefore doesn't have `regeneratorRuntime` defined, leading to an error in file uploads.

I tried working around this by using `import VJsf from "@koumoul/vjsf/lib/VJsfNoDeps";` in the hopes that it would avoid the polyfill, but got an error about require.

To the best of my understanding, the polyfill is not meant for libraries, https://babeljs.io/docs/en/babel-polyfill/ says: `is intended to be used in an application rather than a library/tool.`.

I am not entirely sure what the fix here would be, other than the recommendations given here: https://babeljs.io/docs/en/usage/#polyfill

I was able to work around the issue by explicitly importing the regenerator runtime:
```javascript
import VJsf from "@koumoul/vjsf";
import "@koumoul/vjsf/dist/main.css";
import "regenerator-runtime/runtime"; // Needed to vjsf file upload support
```

**Comment by michalgmazurek** [comment]
Any info about fixing that? Workaround mentioned above did not work for me :( I still receive error 
```
Uncaught ReferenceError: regeneratorRuntime is not defined
    at main.js:1:47777
    at Object../node_modules/@koumoul/vjsf/dist/main.js (main.js:1:48394)```

**Comment by albanm** [comment]
I just released a new version 2.10.2, I think this is solved in it.
